src/Case-Channel.f90

```diff
@@ -93,7 +93,7 @@ subroutine init_channel (ux1,uy1,uz1,ep1,phi1)
                 if (idir_stream == 1) then
                    ux1(i,j,k)=one-y*y
                    uy1(i,j,k)=zero
-                   uz1(i,j,k)=sin(real(i-1,mytype)*dx)+cos(real(k-1,mytype)*dz)
+                   uz1(i,j,k)=zero !sin(real(i-1,mytype)*dx)+cos(real(k-1,mytype)*dz)
```

Comment:
This was probably present to help the transition to turbulence ?

I suggest to remove the commented code or at the very least to replace `k-1` with `k+xstart(3)-2`

Reply:
This should not have been commented

Reply:
The `k-1` was incorrect. We can ship a fix in this PR or in a dedicated one.

Reply:
I have added @mathrack correction for the channel in X. Same should be done also for Z but i think we can do this in a dedicated PR, after testing all channel and relative initialisation.